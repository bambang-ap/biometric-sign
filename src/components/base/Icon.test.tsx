import React from 'react';

import {render} from '@testing-library/react-native';
import FullMockComponent from '__mocks__/@components';
import renderer from 'react-test-renderer';

import {Icon} from './Icon';
describe('Icon Component', () => {
  const TestComponent = () => {
    return (
      <FullMockComponent>
        <Icon name="home" />
      </FullMockComponent>
    );
  };

  test('Icon render correctly', () => {
    render(<TestComponent />);
  });

  test('Icon snapshot', () => {
    const wrapper = renderer.create(<TestComponent />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
