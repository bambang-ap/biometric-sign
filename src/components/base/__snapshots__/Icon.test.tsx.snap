// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Component Icon snapshot 1`] = `
<Text
  allowFontScaling={false}
  selectable={false}
  style={
    [
      {
        "color": undefined,
        "fontSize": 12,
      },
      undefined,
      {
        "fontFamily": "FontAwesome5Free-Solid",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      {
        "fontWeight": "900",
      },
    ]
  }
>
  ï€•
</Text>
`;
